lib/tasks/scripts/clean_biswanath_dupes.rb

```diff
@@ -166,10 +167,15 @@ def deactivate_ambiguous_patients
     ambiguous_matches.each(&:discard_data)
   end
 
+  def log(msg)
+    puts msg if verbose
+  end
```

Comment:
Yup, I like this verbose pattern. This file has been axed 😅 but I will leverage this idea in #969 👍